docs/update_tails_usbs.rst

```diff
@@ -0,0 +1,104 @@
+
+The Tails project releases updates `every 6 to 8 weeks`_. Occasionally they
+release a new version ahead of the normal cycle in order to address a
+security issue. For regular Tails OS and security information, check out the
+Tails `Security page`_ and subscribe to the Tails RSS/Atom feed.
+
+.. _every 6 to 8 weeks: https://tails.boum.org/contribute/calendar/
+.. _Security page: https://tails.boum.org/security/index.en.html
+
+
+Update via Graphical Installer
+------------------------------
+
+For Internet-connected Tails workstations, you'll see a notification when a new version of Tails is available. 
+
+|Update Notification| 
+
+We recommend that you back up your existing configuration and `update as soon as possible`_. It usually takes some time for updates to download, so keep that in mind when choosing when to update.
+
+.. |Update Notification| image:: images/tails_update_notification.png
+.. _update as soon as possible: https://docs.securedrop.org/en/stable/backup_workstations.html
+
+Update Manually
+---------------
+
+The process for manually updating a Tails USB is:
+
+  #. Recommended: :ref:`Make a backup of the USB you want to update <Step 1>`. 
+  #. Install the :ref:`latest version of Tails <Step 2>` on your *primary Tails USB* or a blank USB stick.
+  #. Use the *primary Tails USB* to :ref:`perform a manual update of your desired Tails USB <Step 3>` on a fully offline (airgapped) computer, such as the computer used for the *Secure Viewing Station*.
+
+What you need
+^^^^^^^^^^^^^
+
+  #. Your *Admin Workstation* computer
+  #. A *primary Tails USB* stick (you may still have one; it was used to create the *Admin Workstation*, *Secure Viewing Station*, and *Journalist Workstation* Tails USBs during the initial SecureDrop install process)
+  #. The Tails USB that you want to update
+  #. A Backup USB to back up the data on your existing Tails USBs
+  #. An airgapped computer, such as the computer used for the *Secure Viewing Station*.
+
+.. _Step 1:
+
+1. Back up your Tails USB
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Follow the instructions`_ to back up your existing Tails USB.
+
+.. _Follow the instructions: https://docs.securedrop.org/en/stable/backup_workstations.html
+
+.. _Step 2:
+
+2. Get the latest version of Tails
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have an existing *primary Tails USB*, boot into it on your *Admin Workstation* computer and follow the graphical updater prompts that guide you through the `update process`_.
+
+Alternatively, you can also download and `install the newest version of Tails from scratch`_ (as you did when you first installed SecureDrop). This may be faster if your *primary Tails USB* has not been updated in a while.
+
+.. _update process: https://tails.boum.org/doc/first_steps/upgrade/index.en.html
+
+.. _install the newest version of Tails from scratch: https://docs.securedrop.org/en/stable/set_up_tails.html#install-tails
```

Comment:
Link should be internal rather than to a specific published version of the docs.